app/data/aggregation.ts

```diff
@@ -0,0 +1,68 @@
+export interface CustomAggregation {
+  start: string
+  end: string
+  default: string
+}
+
+export const aggregationList = [
+  {
+    value: '-2',
+    text: '100'
+  },
+  {
+    value: '-1',
+    text: '10'
+  },
+  {
+    value: '0',
+    text: '1'
+  },
+  {
+    value: '1',
+    text: '0.1'
+  },
+  {
+    value: '2',
+    text: '0.01'
+  },
+  {
+    value: '3',
+    text: '0.001'
+  }
+]
+
+const getDecimalPlaceFromValue = (value: string) =>
+  aggregationList.find(({ text }) => text === value)?.value
+
+export const customAggregations = {
```

Comment:
Decided to extract the custom configurations to a separate file for better scalability.